### PR TITLE
Enhancements for the the frontend

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- fixed GitHub logo
- made small hacks that enhanced the work with URL
- added public project checkbox
- updated alignment for navigation bars
- added tips with publicity to projects

### CODE REVIEW

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt [hunk 1]
```diff
@@ -52,6 +47,13 @@ class CollectionView : AbstractView<PropsWithChildren, State>(false) {
                             a(href = "#/${it.value.owner}/${it.value.name}") {
                                 +it.value.name
                             }
+                            span("border ml-2 pr-1 pl-1 text-xs text-muted ") {
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt:54]: Should we extract it into a separate function?

Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt:54]: yes, I was making it right now :)

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/TopBar.kt [hunk 2]
```diff
@@ -87,12 +77,13 @@ fun topBar() = fc<TopBarProps> { props ->
...
                                 if (index == size - 1) {
-                                    a(href = currentLink.removeSuffix("execution")) {
+                                    a {
                                         attrs.classes = setOf("text-warning")
                                         +pathPart
                                     }
                                 } else {
-                                    a(href = currentLink) {
+                                    // small hack to redirect from history/execution to history
+                                    a(currentLink.removeSuffix("execution")) {
```
[the remark this answers is not in the document]
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/TopBar.kt:86]: fixed

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt (hunk 1 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt]: fixed

**Review by petertrr — APPROVED** [review]

**Pull request merged**